src/schema/v2/sale_artwork.ts

```diff
@@ -145,6 +146,32 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
         description: "Singular estimate field, if specified",
         resolve: ({ estimate_cents }) => estimate_cents,
       },
+      formattedStartDateTime: {
+        type: GraphQLString,
+        description:
+          "A formatted description of when the lot starts or ends or if it has ended",
+        resolve: (saleArtwork, _options, { defaultTimezone, saleLoader }) => {
+          if (!!saleArtwork.sale) {
+            return formattedStartDateTime(
+              saleArtwork.sale.start_at,
+              saleArtwork.endedAt ||
```

Comment:
Passing in a waterfall of 3 possible values seems kind of odd, esp since `endedAt` has a different meaning than the others (when set, the lot is closed, no need for any additional check if the current time is past that time).

I think what I'm missing here are examples...aka specs(!)

Reply:
Good question! I think it's ok to keep this specific for cascading end times for now. For instance, can return null if cascading end times aren't set.

But, I can envision a consolidation where the text on a lot page ('Closes X....') is consistent no matter the sale type (in which case we'd want to consider all the use cases).

Not for this PR (fine to just support cascading), but what about something like- Gravity's `SaleArtwork` fields `end_at` and `ended_at` should be backfilled/migrated and set for _all_ sale artworks and all sale types, whether they are cascading or not? In a non cascading end time sale those values would correspond to the sale's values. That feels more consistent from the Gravity side, would that make some of this code (and maybe even Force-side code), easier to keep consistent?

Reply:
Okay i like the idea of returning `null` if not cascading end times and yes, this crossed my mind re: populating `end_at` and `ended_at` either way. I think it's a good idea. Ideally this field (and the field on sale) works for all sales, regardless of whether they have cascade end times. I think at some point maybe once we finish v1 or do the eigen work, we should plan some cleanup for all this.

Reply:
That makes sense re: making these fields just 'work' for all types of sales and lots, agree that's a good follow up.